### ISSUE TITLE
Exception using homeassistant beta

### ISSUE THREAD

**Issue by shaonianzhentan** [vicfergar/HassClient#5]
```c#
Newtonsoft.Json.JsonSerializationException:“Error converting value "2021.12.0b3" to type 'System.Version'. Path 'ha_version', line 1, position 53.”

ArgumentException: Could not cast or convert from System.String to System.Version.
```

![image](https://user-images.githubusercontent.com/5294022/145528259-13036ed9-204d-4a8b-b17f-522f35ec66c8.png)

**Comment by vicfergar** [comment]
Hi @shaonianzhentan,

Good catch! Looks like `System.Version` does not supports letters. I think it would be better to map 'ha_version' into `string` or a custom type.

As a workaround, I recommend you to change it to `string`.

**Comment by shaonianzhentan** [comment]
Thank you for your suggestion. I returned ha to the official version

**Comment by vicfergar** [comment]
It should be fixed on v1.0.11
